flask_dance/consumer/pkce.py

```diff
@@ -0,0 +1,42 @@
+import hashlib
```

Comment:
I think this whole file can be deleted, in favor of the functions provided by `oauthlib`.